### DISCUSSION TITLE
[Tooltip] Don't hide automatic on mobile

### DISCUSSION THREAD

**Discussion by not-SAINT** [epam/UUI#1689] [Q&A] [answered]
After click on an element with Tooltip on mobile, Tooltip stay showed. Until an element has focus/click any other place.
Prop **closeDelay** is ignored.

![image](https://github.com/epam/UUI/assets/60464016/156fd1f5-9f50-4193-9422-a565aaff44b2)

Please, help to understand: this is correct behavior on mobile or a bag?
Thanks.

**Comment by AlekseyManetov** [comment, marked as answer]
In the current moment, we haven't optimized or thoroughly considered how tooltips should work on mobile devices. To me surprise, they work, likely through browser emulations (since mobiles don't have a hover event). 
I think current behavior is sufficient for now. For example, ant design have the same behavior - https://ant.design/components/tooltip. 

Perhaps in the future, we will delve deeper into this topic and enhance mobile behavior, but it's not in our nearest backlog.
